components/progressTracker/ProgressTracker.js

```diff
@@ -0,0 +1,47 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+import theme from './theme.css';
+
+import Box from '../box';
+import ProgressStep from './ProgressStep';
+
+class ProgressTracker extends PureComponent {
+  render() {
+    const { color, children, activeStep } = this.props;
+
+    const classNames = cx(theme['progress-tracker']);
+
+    return (
+      <Box className={classNames}>
+        {React.Children.map(children, (child, index) => {
+          const currentActiveStep = activeStep < 0 ? 0 : activeStep;
+
+          return React.cloneElement(child, {
+            active: index === currentActiveStep,
+            completed: index < currentActiveStep,
+            color: color,
```

Comment:
`color` in ES6 is the same as `color: color`, so you can just get rid of `: color`